### DISCUSSION TITLE
Copying and pasting expressions(curves)?

### DISCUSSION THREAD

**Discussion by yezhiyi9670** [stakira/OpenUtau#1214] [Q&A]
I am trying to cut a part into two. To do this, I have to copy the second half of the part and paste it elsewhere, but the pitch deviation curve does not seem to get copied. Additionally, I cannot find a way to select curves in the piano roll editor and copy them.

I have checked the wiki and searched on the web, but none has mentioned this. The only way I can come up with is to manipulate the `ustx` file directly with a script (which requires closing the editor first).

Is there any intended way to accomplish this in the editor?

(Note: I have discovered the `Alt + V` action, but it does not seem to work with any of these curve expressions)

**Comment by BagelHero** [comment]
Currently, I believe there isn't. There are requests for a part cutting tool (like the note cutting tool), as well as more robust copy and paste tooling, but I believe no current progress on either. Apologies for the inconvenience for right now.
